packages/bus/src/broadcast/types.ts

```diff
@@ -11,6 +11,7 @@
  */
 
 import type { UniqueId } from '@accelint/core';
+import type { StructuredCloneable } from 'type-fest';
```

Comment:
would it be helpful to consumers to re-export this type? Unsure, but curious your thoughts